### PR TITLE
Update settings UI with i18n support，The settings UI supports Chinese.

### PR DESCRIPTION
**Pull request by kuweiguge**
Title：Added Chinese localization to the settings interface

Description： I immensely enjoy using this plugin. However, as a non-native English speaker, I have experienced difficulties in understanding the English text in the settings interface. I am positive that there are other Chinese speaking users who love this plugin as much as I do and might face the same language challenges. So, to eliminate this difficulty, I have added Chinese localization support to enhance the user experience for Chinese users. Looking forward to your feedback and hope this inclusion will be beneficial.

Settings UI:

CleanShot 2023-08-16 at 11 00 06@2x

----

Below is the commit description generated by Idea AI Assistant：

This commit refactors hardcoded strings in `SettingsComponent.java` into `resource bundles`. 
The strings are now extracted into two properties files, `SettingsBundle_en.properties` and `SettingsBundle_zh_CN.properties`. 
This is intended to provide support for internationalization (i18n) and improve code maintainability. 
Also, a new class, `SettingsBundle.java` is created to manage these resource bundles dynamically using the path 'messages.SettingsBundle', which has been added to the plugin.xml file. This refactor creates a reusable and extensible mechanism for localizing the settings UI in the future.

### CODE REVIEW

**Comment by 0verEngineer** [comment]
@kuweiguge Hey, thanks for the PR :)

How did you translate the text? I have checked it with deepL and it looks good.
One thing that i found is this one: "High Red Question Label", that should say Bold, is that an error with deepL?
Screenshot_20230816-102807

**Comment by kuweiguge** [comment]
@0verEngineer Hey
During my previous submission, I utilized the AI assistant from IntelliJ IDEA for Chinese translation, and subsequently noted several inaccuracies. With an aim to enhance translation quality and ensure the validity of the content, I conducted a meticulous examination and made minor adjustments to the translation. I place significant importance on ensuring the translated text isn't just accurate, but also complies with Chinese grammatical norms and context. I am confident that these modifications will make the Chinese context more natural and easier to comprehend. Thank you for considering these further submissions.
CleanShot 2023-08-16 at 17 47 31@2x

File: src/main/java/org/overengineer/inlineproblems/bundles/SettingsBundle.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.PropertyKey;
+
+import java.util.function.Supplier;
+
+/**
+ * The SettingsBundle class is a dynamic bundle for managing internationalized messages for the AiCodeBot project.
+ * It extends the DynamicBundle class.
+ *
+ * @author zhengwei
```
Comment (0verEngineer) [inline src/main/java/org/overengineer/inlineproblems/bundles/SettingsBundle.java:15]: Who is this? Or where does this code come from?

Comment (0verEngineer) [reply src/main/java/org/overengineer/inlineproblems/bundles/SettingsBundle.java:15]: @kuweiguge I cannot merge it without knowing that

Comment (kuweiguge) [reply src/main/java/org/overengineer/inlineproblems/bundles/SettingsBundle.java:15]: **kuwei** is my online nickname, **zhengwei** is the pinyin of my Chinese name, and the code is written by me with reference to the official idea plugin example.

Comment (kuweiguge) [reply src/main/java/org/overengineer/inlineproblems/bundles/SettingsBundle.java]: Refactored author and docstrings in SettingsBundle class

Comment (0verEngineer) [reply src/main/java/org/overengineer/inlineproblems/bundles/SettingsBundle.java]: Perfect thanks a lot, will be merged when i have some time.

**Comment by 0verEngineer** [comment]
@kuweiguge If you have time and interest, it would be cool if you would translate the new strings added here: https://github.com/0verEngineer/InlineProblems/pull/40

**Comment by kuweiguge** [comment]
> @kuweiguge If you have time and interest, it would be cool if you would translate the new strings added here: #40

I'm not quite sure what I should do. Can you guide me? Should I checkout the dev branch, add Chinese translations, and then submit a PR?

**Comment by 0verEngineer** [comment]
Yea submit a PR into dev or main, does not matter. :)

**Pull request merged**